pkg/objstore/oss/oss.go

```diff
@@ -46,6 +46,8 @@ type Bucket struct {
 	bucket *alioss.Bucket
 }
 
+func (b *Bucket) ReloadCredentials() error { return nil }
```

Comment:
I would add this to S3 for now and use optional interface casting to enable reloading. WDYT? It will allow iterating in easy way (:

Reply:
What is optional interface casting 😅 ? @bwplotka